src/bin/club-epics.ts

```diff
@@ -23,14 +24,23 @@ const main = async () => {
     const epics = await client.listEpics();
     spin.stop(true);
     const textMatch = new RegExp(program.title, 'i');
-    epics.filter((epic: Epic) => !!`${epic.name} ${epic.name}`.match(textMatch)).map(printItem);
+    epics
+        .filter((epic: Epic) => {
+            return (
+                !!`${epic.name} ${epic.name}`.match(textMatch) &&
+                !!(program.milestone && epic.milestone_id == program.milestone )
```

Comment:
@adamazing my apologies for not catching this earlier, but this code requires a `milestone` flag currently. As such, you can't get output **without** a `milestone` flag...
I think you'll want to change this to something like
```suggestion
                !!(program.milestone ? epic.milestone_id == program.milestone : true)
```

Reply:
Yikes! Thanks for that good catch. Updated.